packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -385,6 +385,45 @@ export class AccountResolver {
     return await this.cartManager.fetchCartsByUid(account.uid);
   }
 
+  @Features(AdminPanelFeature.AccountSearch)
+  @ResolveField()
+  public async backupCodes(@Root() account: Account) {
+    const uidBuffer = uuidTransformer.to(account.uid);
+    const result = await this.db.recoveryCodes
+      .query()
+      .where('uid', uidBuffer)
+      .resultSize();
+
+    return [
+      {
+        hasBackupCodes: result > 0,
+        count: result,
+      },
+    ];
+  }
+
+  @Features(AdminPanelFeature.AccountSearch)
+  @ResolveField()
+  public async recoveryPhone(@Root() account: Account) {
+    const uidBuffer = uuidTransformer.to(account.uid);
+    console.log(uidBuffer);
+    const result = await this.db.recoveryPhones
+      .query()
+      .select(RECOVERYPHONES_COLUMNS)
+      .where('uid', uidBuffer);
+
+    console.log(result);
+    return [
+      {
+        exists: result.length > 0,
+        phoneNumber:
+          result[0] && result[0].phoneNumber
+            ? result[0].phoneNumber.slice(-4)
```

Comment:
We have lots of PII in admin panel, which is kind of the point. Maybe it's okay to show the full number? Maybe it's safer not show the full number? 

Either way is probably fine, but I just wanted to double check this requirement.

Reply:
Checked in with Joe and he still recommends last four digits. I'll rename for clarity. Thanks!